### ISSUE TITLE
Journal Viewer: Display raw run data

### ISSUE THREAD

**Issue by AdrianPotter** [ISISComputingGroup/IBEX#2903]
As a user, I would like the journal viewer to display the information about each run given below so that I can find out the details about historical runs.

Depends on #2901

Run details:
- Run Number
- Title
- Start Time (YYYY-MM-DD HH:MM:SS)
- Duration (HH:MM:SS)
- uAmp Hours (up to 7 significant figures)
- RB Number
- Users

This ticket depends on the basic architecture having been set up in #2901

Notably the intent here is to make sure the data is read and available. Formatting is deferred to a separate ticket.

## Acceptance criteria
- [ ] The current run data stored in the database is available in the journal viewer
- [ ] The data is never more than 10s out of date
- [ ] The fields for each run correspond to those given in the ticket
- [ ] The functionality works in E3 and E4
- [ ] The results correspond to the current instrument the GUI is pointing at.
